bip-0047.mediawiki

```diff
@@ -85,7 +85,7 @@ This derivation level is equivalent to the Account level in BIP-44. Wallets SHOU
 
 For example, the payment code created represented by (m / 47' / 0' / 0') is part of the account represented by (m / 44' / 0' / 0').
 
-The second account in a wallet consists of the new account/payment code pair created by using an index of 1 in as the account/identity level of both paths.
+The second account in a wallet consists of the new account/payment code pair created by using an index of 1 used as the account/identity level of both paths.
```

Comment:
I'm not sure if this change reflects the author's intent. It looks to me like the author's intent was either "in" or "as" without the other, but would need to look further or ask the author.